### ISSUE TITLE
Titlecase vs. Uppercase

### ISSUE THREAD

**Issue by whyisjake** [WordPress/wp-autoupdates#120]
I think we are being pretty deliberate, but I think we might want to change a few things to enforce uppercase (@audrasjb commented about how all Americans want to capitalize everything...)

<img width="821" alt="Screen Shot 2020-05-07 at 10 36 21 AM" src="https://user-images.githubusercontent.com/848187/81326844-46a11780-904f-11ea-82f7-ecd674452cc3.png">

👆This does seem to follow [titlecase](http://titlecase.com/), but I think we should be using uppercase for these string. Looks better in the prominent area of the page.

<img width="267" alt="Screen Shot 2020-05-07 at 10 36 28 AM" src="https://user-images.githubusercontent.com/848187/81326851-47d24480-904f-11ea-86e7-6a424049256f.png">

☝️Kinda feel the same here. This isn't a sentence, I almost wonder if this should be a button or dropdown to ensure that people know this is a UI element. The JS actions here can be almost invisible, so ensuring that this looks and behaves like a button might be in our best interest. Toggle? Radio? Checkbox???

<img width="238" alt="Screen Shot 2020-05-07 at 10 39 53 AM" src="https://user-images.githubusercontent.com/848187/81326853-486adb00-904f-11ea-9082-03fbeccbbbc3.png">

☝️ Bulk Actions is the headline for the dropdown, but I think we should follow and make that string uppercased too.

**Comment by audrasjb** [comment]
I'd personally use sentence case, but… why not.
In any case, we need to fix this inconsistency :)

**Comment by whyisjake** [comment]
I wonder if we should use this UI element for the table view.

<img width="267" alt="Screen Shot 2020-05-07 at 11 26 44 AM" src="https://user-images.githubusercontent.com/848187/81331298-3b051f00-9056-11ea-8bf3-ea3bcdc4ad78.png">

**Comment by whyisjake** [comment]
☝️could be auto-updates disabled

👇 could be auto-udpates enabled

<img width="133" alt="Screen Shot 2020-05-07 at 11 32 15 AM" src="https://user-images.githubusercontent.com/848187/81331413-6a1b9080-9056-11ea-9a68-ac5734eb7e46.png">

**Comment by pbiron** [comment]
With regard to buttons vs links, I don't know if the same reasoning should apply here (tho I suspect it would), but prior to 5.4 the Privacy tools (which are also list tables) had buttons in their "Next Steps" column.  Those were changed to links (visually) in 5.4 on recommendation from the design team.

See trac ticket [#49323](https://core.trac.wordpress.org/ticket/49323).

**Comment by pbiron** [comment]
@audrasjb I just did some checking and if using `<button class="button-link">` would be better for a11y (which is what the Privacy tools mentioned above use) we can switch to that with only minor mods to [wp_autoupdates_plugins_enabler()](https://github.com/WordPress/wp-autoupdates/blob/9cf0371324ee32dddc28cc6d5b31cf23b12bcbe2/functions.php#L319) and [wp_autoupdates_themes_enabler()](https://github.com/WordPress/wp-autoupdates/blob/9cf0371324ee32dddc28cc6d5b31cf23b12bcbe2/functions.php#L361) (for non JS-enabled browsers) and the click handler in the JS for Ajax.

It is pretty late in the game to be making a change like that, but if it is better for a11y, I suspect it will also come up on the core merge patch.  So, we might as well do it in the plugin as well.

**Comment by whyisjake** [comment]
Yeah, I don't think this should be a blocker for the core merge, but should at least be brought up and get `design-feedback` on. Thanks for the links @pbiron. Great context.

**Comment by wittwitsan** [comment]
There is an audit /wp-admin ticket [#49616](https://core.trac.wordpress.org/ticket/49616) about sentence-case UI elements.
